addons/l10n_ar_website_sale/controllers/main.py

```diff
@@ -0,0 +1,57 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import _
+from odoo.addons.website_sale.controllers.main import WebsiteSale
+from odoo.http import request, route
+
+
+class L10nARWebsiteSale(WebsiteSale):
+
+    def _get_mandatory_billing_fields(self):
+        """Extend mandatory fields to add new identification and responsibility fields when company is argentina"""
+        res = super()._get_mandatory_billing_fields()
+        if request.website.sudo().company_id.country_id.code == "AR":
+            res += ["l10n_latam_identification_type_id", "l10n_ar_afip_responsibility_type_id", "vat"]
+        return res
+
+    @route()
+    def address(self, **kw):
+        """Extend to send information about the identification types and AFIP responsibility to show in the address form"""
+        response = super().address(**kw)
+        if request.website.sudo().company_id.country_id.code == "AR":
+            response.qcontext.update({'identification_types': request.env['l10n_latam.identification.type'].search(['|', ('country_id', '=', False), ('country_id.code', '=', 'AR')]),
+                                      'responsibility_types': request.env['l10n_ar.afip.responsibility.type'].search([]),
+                                      'identification': kw.get('l10n_latam_identification_type_id'),
+                                      'responsibility': kw.get('l10n_ar_afip_responsibility_type_id')})
```

Comment:
@Bruno-Zanotti The security checks on this branch failed because of this (using the qcontext).  Looking deeper in to it, it seems really hacky as in the super the template has already been rendered.